tests/system/test_pandas.py

```diff
@@ -41,12 +45,12 @@
     "google.cloud.bigquery_storage", minversion="2.0.0"
 )
 
-if pandas is not None:
-    PANDAS_INSTALLED_VERSION = pkg_resources.get_distribution("pandas").parsed_version
-else:
-    PANDAS_INSTALLED_VERSION = pkg_resources.parse_version("0.0.0")
+if pandas is not None:  # pragma: NO COVER
+    PANDAS_INSTALLED_VERSION = metadata.version("pandas")
+else:  # pragma: NO COVER
+    PANDAS_INSTALLED_VERSION = "0.0.0"
 
-PANDAS_INT64_VERSION = pkg_resources.parse_version("1.0.0")
+PANDAS_INT64_VERSION = "1.0.0"
```

Comment:
```suggestion
```


Instead of 
```
@pytest.mark.skipif(
    PANDAS_INSTALLED_VERSION < PANDAS_INT64_VERSION,
    reason="Only `pandas version >=1.0.0` is supported",
)
```
we could have
```
@pytest.mark.skipif(
    PANDAS_INSTALLED_VERSION[0:2] in ["0."],
    reason="Only `pandas version >=1.0.0` is supported",
)
```